modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase

```diff
@@ -0,0 +1,269 @@
+			EVPOrganization.viewOptionNotPresentInKebabMenu(
+				menuItem = "Review",
+				organizationName = "Organization Test");
+		}
+	}
+
+	@description = "Story ID: LPS-164733 | Test Summary: Verify that Only the Finance user can not see 'Review button' when organization are in 'Awaiting Approval On EVP' status"
+	@priority = 5
+	test FinanceCanNotSeeReviewInAwaitingEVPStatus {
+		task ("Given that the Manager user create a organization") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "manager@liferay.com",
+				userLoginFullName = "Manager User");
+
+			EVPUtils.gotoEVP();
+
+			EVPOrganization.goToCreateOrganization();
+
+			EVPOrganization.fillOrganizationForm(
+				bankingInfo = "Bank Test",
+				city = "City Test",
+				contactEmail = "test@liferay.com",
+				contactName = "Contact Test",
+				contactPhone = 12345,
+				country = "Country Test",
+				organizationName = "Organization Test",
+				organizationSite = "www.test.com.br",
+				smallDescription = "Description Test",
+				state = "State Test",
+				street = "Street Test",
+				taxId = 12345,
+				zip = "12.345-67");
+		}
+
+		task ("When the finance user go to organization created") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "finance@liferay.com",
+				userLoginFullName = "Finance User");
+
+			EVPUtils.gotoEVP();
+
+			EVPNavigator.goToNavItem(navItem = "Organization");
+		}
+
+		task ("Then the finance user can not see Review option in kebab menu") {
+			EVPOrganization.viewOptionNotPresentInKebabMenu(
+				menuItem = "Review",
+				organizationName = "Organization Test");
+		}
+	}
+
+	@description = "Story ID: LPS-164733 | Test Summary: Verify that the EVP Staff can approve a new organization"
+	@priority = 4
+	test ManagerCanApproveOrganization {
+		task ("Given that Sign as Manager user and Navigate to EVP Site") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "manager@liferay.com",
+				userLoginFullName = "Manager User");
+
+			EVPUtils.gotoEVP();
+		}
+
+		task ("And that the manager create a organization") {
+			EVPOrganization.goToCreateOrganization();
+
+			EVPOrganization.fillOrganizationForm(
+				bankingInfo = "Bank Test",
+				city = "City Test",
+				contactEmail = "test@liferay.com",
+				contactName = "Contact Test",
+				contactPhone = 12345,
+				country = "Country Test",
+				organizationName = "Organization Test",
+				organizationSite = "www.test.com.br",
+				smallDescription = "Description Test",
+				state = "State Test",
+				street = "Street Test",
+				taxId = 12345,
+				zip = "12.345-67");
+		}
+
+		task ("When the manager approve the organization created") {
+			EVPOrganization.kebabMenuOrganization(
+				kebabOption = "Review",
+				organizationName = "Organization Test");
+
+			EVPOrganization.reviewOrganization(
+				button = "Approve",
+				typeMessage = "Your organization has been approved!");
+		}
+
+		task ("Then can view the status Awaiting Finance Approval") {
```

Comment:
Hi @joelrocha68, Could you please rename this task to `Then the status will be updated to 'Awaiting Finance Approval'`? Just to be more clear.